lace/archive/nyx_archive.py

```diff
@@ -41,7 +41,9 @@ def __init__(self, file_name=None, kp_Mpc=0.7):
         # list all simulations
         self.list_sim = self.list_sim_cube + self.list_sim_test
         ## done set simulation list
-
+        
+        # list of available redshifts, temporary hack, chech higher redshfits
+        self.list_sim_redshifts = [str(d) for d in np.round(np.arange(2.2,4.7,0.2),2) ]
```

Comment:
@lauracabayol - this hard-coded list of redshifts is not great, since some sims have z=2.0 and most of them (all?) also go beyond z=4.6... do you need it?

Reply:
I see that you are aware that this is a hack, but do you want to put it to the main branch?

Reply:
For now, we can limit the Nyx archive to this redshift range, and only go outside this range after Durham.

Reply:
I made some changes to cover all possible Nyx redshifts, from z=2 to 5.4, skipping 4.8 and 5.2